python/paddle/incubate/autograd/composite_rules.py

```diff
@@ -171,6 +171,36 @@ def layernorm_composite(x, scale, bias, epsilon, begin_norm_axis):
     return out, mean_, variance
 
 
+@REGISTER_COMPOSITE('instance_norm')
+def instancenorm_composite(x, scale, bias, epsilon):
+    """
```

Comment:
refer to layer_norm, add amp-relative process.

Reply:
done